### PR TITLE
f(obstacle_avoidance) initial spec

### PR DESCRIPTION
**Pull request by SammyOina**

### CODE REVIEW

File: include/confs.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#ifndef CONNFS_h
```
Comment (WashingtonKK) [inline include/confs.h:1]: `CONNFS` or `CONFS`

File: include/confs.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#define MOTOR_BASE_SPEED_LEFT 80
+#define MOTOR_BASE_SPEED_RIGHT 104
+
+int line_position;
+
+void Drive60LeftRoutine(Satima leftDriver, Satima rightDriver){
+    leftDriver.setSpeed(104, CLOCKWISE);
+    rightDriver.setSpeed(20, CLOCKWISE);
+    delay(20);
+    leftDriver.setSpeed(104, CLOCKWISE);
```
Comment (WashingtonKK) [inline include/confs.h]:
```suggestion
    leftDriver.setSpeed(90, CLOCKWISE);
```

File: include/confs.h [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    rightDriver.setSpeed(80, CLOCKWISE);
```
Comment (WashingtonKK) [inline include/confs.h]:
```suggestion
    rightDriver.setSpeed(90, CLOCKWISE);
```

File: include/confs.h [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    delay(20);
```
Comment (WashingtonKK) [inline include/confs.h:15]: Add delay constant
```suggestion
    delay(DELAY_CONSTANT);
```

**Review by WashingtonKK — APPROVED** [review]

**Review by kimanimichael — APPROVED** [review]

**Pull request merged**